src/ripple/core/impl/Config.cpp

```diff
@@ -481,6 +481,9 @@ Config::loadFromString(std::string const& fileContents)
     if (getSingleSection(secConfig, SECTION_COMPRESSION, strTemp, j_))
         COMPRESSION = beast::lexicalCastThrow<bool>(strTemp);
 
+    if (getSingleSection(secConfig, SECTION_LEDGER_REPLAY, strTemp, j_))
+        LEDGER_REPLAY = beast::lexicalCastThrow<bool>(strTemp);
```

Comment:
I think the idea is that the option in config is temporary, since the feature is experimental. This feature is a protocol improvement and not user facing. Once everyone feels comfortable about it, it will become part of a protocol version, like `ValidatorListPropagation`. And the feature negotiation of the peer connection handshake will use the protocol versions instead of configurations. 

That was my understanding. So I don't expect the section expands. But if you think there are more useful configurations, we can add them.